### PR TITLE
Added decision forest model w/ tuner for hyperparameters, generated models

### PR DESCRIPTION
**Pull request by cameronmoran23**
# Overview

**Type of Change:** New Feature

**Summary:** Updated the notebook to train a decision forest w/ hyperparameter tuning

## UI/UX Implementation
No changes made

## Model Updates
No changes made

### Data Models
No changes made

### Object-Oriented Models
No changes made

## End-to-End Testing Instructions
Run all in the jupyter notebook to train and save the models

### CODE REVIEW

**Comment by cameronmoran23** [comment]
[Test a variety of hyperparameters with our tensorflow model to try to improve the accuracy](https://trello.com/c/6UkVUUXH/85-test-a-variety-of-hyperparameters-with-our-tensorflow-model-to-try-to-improve-the-accuracy)

**Comment by cameronmoran23** [comment]
[Test a tensorflow decision forest classifier to possibly improve accuracy](https://trello.com/c/xTbd2A4d/87-test-a-tensorflow-decision-forest-classifier-to-possibly-improve-accuracy)

**Review by DILLON9999 — APPROVED** [review]
Good practice to separate each model into its own folder
